src/main/resources/nwoolcan/view/step_detail.fxml

```diff
@@ -36,54 +38,65 @@
                             <VBox>
                                 <VBox>
                                     <HBox>
-                                        <Label text="Step type :" />
-                                        <Label fx:id="stepTypeNameLabel" />
+                                        <Label text="Step type :"/>
```

Comment:
use grid pane, its much more better

Reply:
1) It's not my code, i just modified the logic of showing chart in the vbox
2) If it is doing it's job I don't see why I should use another pane